account_payment_purchase/views/purchase_order_view.xml

```diff
@@ -9,11 +9,7 @@
         <field name="inherit_id" ref="purchase.purchase_order_form" />
         <field name="arch" type="xml">
             <field name="payment_term_id" position="after">
-                <field
-                    name="payment_mode_id"
-                    domain="[('payment_type', '=', 'outbound')]"
-                    widget="selection"
-                />
+                <field name="payment_mode_id" widget="selection" />
```

Comment:
AFAIK, this widget is conflicting in certain cases in v16. Better to use m2o

Reply:
out of curiosity do you have some references or more details about the problems?

Reply:
#1125

Reply:
done